torch/fx/proxy.py

```diff
@@ -153,6 +155,9 @@ def __bool__(self) -> bool:
     def keys(self):
         return self.tracer.keys(self)
 
+    def __len__(self):
+        raise RuntimeError("'builtins.len' is not supported. Replace `builtins.len` with 'torch.fx.len' and try again.")
```

Comment:
Tiny nit: typically folks are going to have `len()` in their code rather than `builtins.len()`, so I think the error message here should just mention `len` without the `builtins` prefix